source/index.haml

```diff
@@ -1,8 +1,7 @@
 ---
-header_title: Aptible Support
```

Comment:
Removing this is causing the index spec to fail because it removes the large 'Aptible Support' header on the index page.